src/calculateRentalCost.js

```diff
@@ -29,6 +29,28 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const SumaOneDay = 40;
```

Comment:
Variable names should be written in camelCase. 'SumaOneDay' should be 'sumOneDay'.